app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= "#{@recipe.name}" %></h1>
+<p><%= "Complexity: #{@recipe.complexity}" %></p>
+<p><%= "Genre: #{@recipe.genre}" %></p>
+<ul><p><%= "Ingredients:" %></p>
+  <% @recipe.ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
+  <% end %>
+</ul>
+
+<h4><%= "Total Cost: #{@recipe.ingredients.total_cost}" %></h4>
```

Comment:
Since you're calling `total_cost` on this collection, it should end up showing the right amounts on this page, but be careful with this kind of design because it could also be called on "all" ingredients which would give you a different amount. As long as this is tested thoroughly, you should be fine.